projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+# Reuse places & organizers
+
+When creating an event via the UiTdatabank Entry API, you can and must use existing places (locations) and organizers in UiTdatabank.
+
+By reusing existing places and organizers you save yourself work, as you don't have to create the place or organizer yourself. In addition, you ensure that no duplicates are created in UiTdatabank and that other integrators can also use the same unique and clean dataset.
+
+Only in the case a certain place or organizer does not exist yet, a new one can be created.
+
+<!-- theme: warning -->
+
+> When sending event content to UiTdatabank trough the Entry API it is essential to reuse existing organizer IDs and place IDs. Otherwise your integration can be blocked because of the creation of duplicates.
+
+On this page, we outline some options to reuse existing places & organizers from UiTdatabank in your integration.
+
+## Reuse existing places
+
+### Storing the place ID(s) inside your application
+
+This is the simplest method. You store the identifier(s) of the place(s) inside your application.
+
+* ✅ Suitable when all the events created through your application all have the same location
+* ✅ Suitable when the events created through your application can have a different location, but (1) the amount of possible locations is limited & (2) the list of possible locations is static and won't change a lot over time
+* ❌ Not suitable when the locations of the events created through your application are unpredictable in advance
+
+#### Example
+
+In the example below, all the events created through the integration take place in "Het Depot", a concert hall in Leuven. First you'll need to find the identifier of the place `Het Depot` in UiTdatabank.
+
+**request**
+
+Perform a `GET` request to the UiTdatabank Search API, including the name of the place and the postal code:
+
+```
+GET /places/?q=name.\*:"Het Depot" AND address.\*.postalCode:3000
+```
+
+**response**
+
+The response now contains the identifier of "Het Depot":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/place/8248e289-c986-4006-902f-b0616dcbcde7",
+         "@type":"Place"
+      }
+   ]
+}
+```
+
+You can now store the identifier of the place "Het Depot" (`8248e289-c986-4006-902f-b0616dcbcde7`) in your application to reuse it as location ID in the `POST` request of the event(s) that you want to create.
```

Comment:
```suggestion
You can now store the URL of the place "Het Depot" (`https://io.uitdatabank.be/place/8248e289-c986-4006-902f-b0616dcbcde7`) in your application to reuse it as `location.@id` in the `POST /events` request of the event(s) that you want to create.
```

(Note that the `POST /events` endpoint that you referred to is documented to use `location.@id` with a URL, unless you specifically switch to the "deprecated" body: https://docs.publiq.be/docs/uitdatabank/73ffd1ab81f76-event-create)